.github/workflows/main.yaml

```diff
@@ -69,7 +69,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, '3.10']
```

Comment:
Should all of them have `''`?

Reply:
It's only needed for `3.10` because YAML converts/rounds it to `3.1`. 😩 

Might as well do the lot though!

Reply:
✔️

Reply:
lol, classic YAML. Thanks Sasha.